.github/workflows/rust.yml

```diff
@@ -79,8 +79,35 @@ jobs:
     - name: Setup Python
       uses: actions/setup-python@v2
 
-    - name: Install numpy and  flit
-      run: pip3 install numpy flit prettytable
+    - name: Install common Python dependencies
+      run: pip3 install numpy flit prettytable wheel
+
+    - name: Cache Apache TVM
+      id: tvm-cache
+      uses: actions/cache@v2
+      with:
+        path: frontends/relay/tvm
+        key: tvm-bin-${{ runner.os }}-${{ steps.versions.outputs.tvm }}
```

Comment:
This will not work. The `versions` step above has to generate a version for TVM and set the value for `tvm`. See how we do it for `verilator`.

Also validate that the caching is working by running a build with no significant changes and seeing if the TVM step is correctly skipped.

Reply:
I just forgot to add a version for the key, so `steps.versions.outputs.tvm` == `""`. Caching works: [actions](https://github.com/cucapra/calyx/runs/1846403476?check_suite_focus=true).